Dockerfile

```diff
@@ -1,22 +1,22 @@
 #
 # Dockerfile for Storm Mesos framework
 #
-FROM mesosphere/mesos:0.22.0-1.0.ubuntu1404
+FROM mesosphere/mesos:0.25.0-0.2.70.ubuntu1404
 MAINTAINER Timothy Chen <tnachen@gmail.com>
 
 # build packages
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update
-RUN apt-get install -yq openjdk-7-jdk
+RUN apt-get install -yq openjdk-7-jdk maven wget
 
 # export environment
 ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
 ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
 
-# copy local checkout into /opt/storm
-ADD . /opt/storm
+ADD . /work
 
+WORKDIR /work
+RUN ./bin/build-release.sh main
```

Comment:
If, for some reason, I wanted to run a shell inside the container I would execute:

```shell
docker run -i --net=host -t brndnmtthws/storm /bin/bash
```

Where `/bin/bash` above is the command I'm running inside the container (i.e., the entrypoint). It's a little different from specifying an explicit entrypoint (either via the Dockerfile, or with --entrypoint).

The tl;dr is that you don't _need_ to specify an entrypoint or command in the Dockerfile. In fact, I prefer _not_ specifying it. That kind of metadata belongs in the Marathon definition, rather than the container image, IMO.